packages/tasks/src/snippets/js.ts

```diff
@@ -41,6 +41,35 @@ for await (const chunk of inference.chatCompletionStream({
 		return snippetBasic(model, accessToken);
 	}
 };
+
+export const snippetImageTextToTextGeneration = (model: ModelDataMinimal, accessToken: string): string => {
+	if (model.tags.includes("conversational")) {
+		// Conversational model detected, so we display a code snippet that features the Messages API
+		return `import { HfInference } from "@huggingface/inference";
+
+const inference = new HfInference("${accessToken || `{API_TOKEN}`}");
+const image_url = "https://cdn.britannica.com/61/93061-050-99147DCE/Statue-of-Liberty-Island-New-York-Bay.jpg"
```

Comment:
```suggestion
const image_url = "https://cdn.britannica.com/61/93061-050-99147DCE/Statue-of-Liberty-Island-New-York-Bay.jpg";
```

nit: ending `;` is missing

Reply:
addressed in https://github.com/huggingface/huggingface.js/pull/927/commits/d9d5a4353d5ccf8c4fe47fe92217a4e869cecb48